pkg/cost/calculator_test.go

```
@@ -0,0 +1,408 @@
+			},
+			SIGUsages: []cost.SIGUsage{
+				{Name: "compute", CPUPercent: 20.0, MemPercent: 15.0, RunCount: 40},
+			},
+			TopPRs: []cost.PRUsage{
+				{PR: "100", Repo: "kubevirt", Org: "kubevirt", CPUPercent: 5.0, MemPercent: 3.0, RunCount: 4},
+			},
+		}
+
+		tmpDir, err := os.MkdirTemp("", "cost-report-test-*")
+		Expect(err).NotTo(HaveOccurred())
+		defer os.RemoveAll(tmpDir)
```

golangci-lint (2.12.2)

[error] Error return value of `os.RemoveAll` is not checked

(errcheck)